dandiapi/api/migrations/0021_productionapplication.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.10 on 2021-12-30 21:19
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import oauth2_provider.generators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('api', '0020_zarr'),
+    ]
```

Comment:
This should look like https://github.com/dandi/dandi-api/blob/74d80f6855b34c09e338b61dc3fc657d07e0d632/dandiapi/api/migrations/0011_stagingapplication.py#L13-L15 instead so that it is available for oauth-toolkit, then you would need another "dummy" migration that has this migration as a dependency (see migration 0012).

Reply:
Ok, I'll need your help in getting that right, but let's wait until after embargo lands, as you said.